evalml/pipelines/components/transformers/scalers/standard_scaler.py

```diff
@@ -16,3 +21,13 @@ def __init__(self, random_state=0, **kwargs):
         super().__init__(parameters=parameters,
                          component_obj=scaler,
                          random_state=random_state)
+
+    def transform(self, X, y=None):
```

Comment:
This is necessary because scikit-learn's StandardScaler's transform method doesn't take in a y, so trying to pass it will error.

https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.StandardScaler.html#sklearn.preprocessing.StandardScaler.transform